packages/vant/src/tabs/utils.ts

```diff
@@ -1,11 +1,16 @@
-import { raf } from '@vant/use';
+import { raf, cancelRaf } from '@vant/use';
 import { ScrollElement, getScrollTop, setScrollTop } from '../utils';
 
+let scrollLeftToRafId: number;
+let scrollTopToRafId: number;
```

Comment:
If we use a global variable to store the id, it may break the scroll animation if there are multiple components. So we should store the id at the component level.

Reply:
Hmm, I didn't consider the case where multiple components exist at the same time. I will fix it later.